Library/Homebrew/cmd/fetch.rb

```diff
@@ -1,3 +1,23 @@
+#:  * `fetch` [`--force`] [`-v`] [`--devel`|`--HEAD`] [`--deps`] [`--build-from-source`|`--force-bottle`] <formulae>:
+#:    Download the source packages for the given <formulae>.
+#:    For tarballs, also print SHA-1 and SHA-256 checksums.
```

Comment:
It should be picked up by the `brew man` command, but I forgot to run it after this fix. Have done so now.